src/runtime-rs/crates/hypervisor/src/kernel_param.rs

```diff
@@ -7,6 +7,8 @@
 use anyhow::{anyhow, Result};
 
 use crate::{VM_ROOTFS_DRIVER_BLK, VM_ROOTFS_DRIVER_PMEM};
+use crate::{VM_ROOTFS_FILESYSTEM_EROFS, VM_ROOTFS_FILESYSTEM_EXT4, VM_ROOTFS_FILESYSTEM_XFS};
+use crate::{VM_ROOTFS_ROOT_BLK, VM_ROOTFS_ROOT_PMEM};
```

Comment:
The two `use`s merged into one line `use crate::{...}` will be better.

Reply:
@Apokleos Thanks for your review! I merged use crate.